<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Resh" failures="2" tests="4" name="Test" time="157.88" errors="0" timestamp="2024-11-13T16:53:37 IST">
  <testcase classname="com.Login.LoginPage" name="Login_Function" time="47.476"/>
  <testcase classname="com.Login.LoginPage" name="Login_Function" time="27.031">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at com.Login.LoginPage.Login_Function(LoginPage.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- Login_Function -->
  <testcase classname="com.Login.LoginPage" name="Login_Function" time="59.439"/>
  <testcase classname="com.Login.LoginPage" name="Login_Function" time="20.326">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at com.Login.LoginPage.Login_Function(LoginPage.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- Login_Function -->
</testsuite> <!-- Test -->
